oak_abi/src/label/tests.rs

```diff
@@ -101,10 +101,10 @@ fn label_flow() {
     //       label_0    label_1
     //             \     /
     //              \   /
-    //          public_trusted
+    //          public_untrusted
```

Comment:
but public_untrusted isn't the bottom label.

Reply:
Not even in the context of the 5 labels of this example? This is not meant to be an exhaustive diagram :)

Reply:
Ooopsies you are right.